### PR TITLE
Changed flags from five_thousand_flag to has_raised_funds

### PR DESCRIPTION
**Pull request by jontours**
Flags changed, ready for review.

### CODE REVIEW

File: openfecwebapp/templates/partials/candidates-filter.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       <ul>
         <li>
           <input id="cycle-checkbox-five-thousand" name="five_thousand_flag" type="checkbox" value="true">
```
Comment (noahmanger) [inline openfecwebapp/templates/partials/candidates-filter.html]: This `name` attribute needs to be changed to `has_raised_funds` too.

Comment (ccostino) [inline openfecwebapp/templates/partials/candidates-filter.html]: The `name` attribute needs to change to `has_raised_funds` as well, and I would also change the `id` attribute to reflect this (so make it `cycle-checkbox-has-raised-funds`).

File: openfecwebapp/templates/partials/candidates-filter.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
...
-          <label for="cycle-checkbox-five-thousand">Has raised more than $5,000</label>
+          <label for="cycle-checkbox-five-thousand">Has raised more than $0</label>
```
Comment (ccostino) [inline openfecwebapp/templates/partials/candidates-filter.html]: In addition to the above change, the `for` attribute should change as well (`cycle-checkbox-has-raised-funds`).

**Comment by ccostino** [comment]
Thanks, @jontours!

**Pull request merged**